recce/apis/check_api.py

```diff
@@ -15,6 +15,7 @@ class CreateCheckIn(BaseModel):
     name: Optional[str] = None
     description: str = ''
     run_id: Optional[str] = None
+    model_id: Optional[str] = None
```

Comment:
please add 

```
    type: RunType
    params: Optional[dict] = None
```
instead of model_id

Reply:
change to use `node_id` and `type` has become a required parameter